dpnp/backend/custom_kernels_random.cpp

```diff
@@ -382,8 +381,8 @@ void custom_rng_standard_normal_c(void* result, size_t size)
         return;
     }
 
-    const _DataType mean =  _DataType(0.0);
-    const _DataType stddev =  _DataType(1.0);
+    const _DataType mean = _DataType(0.0);
+    const _DataType stddev = _DataType(1.0);
```

Comment:
It is better to make style amendments not related to this functionality in a separate PR.